charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-snapshot-validation-webhook-deployment.yaml

```diff
@@ -19,7 +19,6 @@ spec:
 {{- end }}
       labels:
         app: snapshot-validation
-        networking.gardener.cloud/from-shoot-apiserver: allowed
```

Comment:
without this label,  can shoot api server pod call csi snapshot validation webhook in shoot control plane namespace?

Reply:
This is ensured by https://github.com/gardener/gardener-extension-provider-alicloud/commit/3f1780cc1f51c57f83007de521839bb003b4b296#diff-cc0552e76e65ffff14fa4d53e143f0d325fb62ff3c926693a9153658dcbfd23dR55 now.